core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -520,7 +520,8 @@ features_list(N) ->
     sets:to_list(sets:from_list(kz_json:get_keys(features(N)))).
 
 -spec set_features(knm_phone_number(), kz_json:object()) -> knm_phone_number().
-set_features(N, Features=?JSON_WRAPPER(_)) ->
+set_features(N, Features) ->
+    'true' = kz_json:is_json_object(Features),
```

Comment:
Isn't pattern checking a macro value way less expensive? What is wrong with the macro?
Also if you remove the macro please remove the now useless include_lib(kz_json.hrl).